docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -182,6 +182,17 @@ You can also create global `HttpSecurityPolicy` invoked on every request.
 Just do not implement the `io.quarkus.vertx.http.runtime.security.HttpSecurityPolicy.name` method and leave the policy nameless.
 ====
 
+[[policy-active-cdi-request-context]]
+=== Inject `@RequestScoped` beans into HttpSecurityPolicy
```

Comment:
Does this content apply to `HttpSecurityPolicy` only ?

Reply:
yes (as you know, the policy are both configurable in app props and by custom beans, but yes) only for the policy; it is about WHEN it is running, it is exception

Reply:
of course there is some text that is general, but I had to introduce the topic